charts/nexus-iq/templates/test/test-connection.yaml

```diff
@@ -1,14 +1,13 @@
 apiVersion: v1
```

Comment:
Discovered this file already existed, but it wasn't working, so I am updating the test, and adding a new one to check the logs.